embedded/servo_bq/set_servo_message.hh

```diff
@@ -1,13 +1,17 @@
 #pragma once
 
 #include "infrastructure/comms/schemas/message.hh"
+// #include "infrastructure/comms/serialization/serialization_macros.hh"
+#include <vector>
 
 namespace jet {
 
 struct SetServoMessage : Message {
-  float target_angle; //TODO units
+  std::vector<int> servo_indices;
+  std::vector<float> target_radians;
```

Comment:
Suggest: 
```
struct ServoCommand {
    int id;
    double target_rads;
    SERIALIZABLE_STRUCTURE(ServoCommand, id, target_rads)
};
```

```
struct SetServoMessage : Message {
    std::vector<ServoCommand> servo_commands;
    MESSAGE(SetServoMessage, servo_commands);
};
```

That way, the indices and commands are guaranteed to be correspondent

Reply:
Ben and I tried for a while earlier today to get that to work but there is a vector<struct> serialization issue we couldn't figure out in time.

Reply:
Alright